connector_importer_source_sftp/tests/test_event_listener.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# @author: Simone Orsi <simahawk@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import base64
+import io
+
+import mock
+
+from odoo.tools import mute_logger
+
+from .common import SFTPSourceSavepointComponentRegistryCase
+
+MOD_PATH = "odoo.addons.connector_importer_source_sftp"
+EVENT_LISTENER_PATH = (
+    MOD_PATH + ".components.event_listeners.SFTPSourceImportRecordsetEventListener"
+)
+SOURCE_MODEL_PATH = MOD_PATH + ".models.source_csv_sftp.ImportSourceCSVSFTP"
+
+
+class TestRecordImporterFinishedEvent(SFTPSourceSavepointComponentRegistryCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.fake_lines = cls._fake_lines(cls, 10, keys=("id", "fullname"))
+        cls.recordset.write(
+            {"source_model": "import.source.csv.sftp", "source_id": cls.source.id}
+        )
+
+    def setUp(self):
+        super().setUp()
+        # Not sure why but this write gets lost if done in setUpClass :/
+        # Probably depends on server env override?
+        # self.source.write(
+        #     {
+        #         "sftp_path_input": "input",
+        #         "sftp_path_error": "error",
+        #         "sftp_path_success": "success",
+        #     }
+        # )
```

Comment:
`setupClass` or `setUp`? Is it still relevant? As it is commented and I can't see any piece of code updating these fields for the test?

Reply:
cleaning